.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+.idea/
```

Comment:
No, it's more "set it and forget it". Check out the link from my first comment. You just keep a file on your computer someplace that looks like a gitignore file and you put its location in your configuration for git (can be done from cli too).